README.md

```diff
@@ -277,6 +278,138 @@ service call. If the camera is not in `Connected` state the driver will attempt
+calibration objects. This is not needed for regular (6-DOF) calibration.
+
+### hand_eye_calibration/load
+[zivid_interfaces/srv/HandEyeCalibrationLoad.srv](./zivid_interfaces/srv/HandEyeCalibrationLoad.srv)
+
+Loads a working directory from a previous hand-eye calibration session. See the
+[hand_eye_calibration/start](#hand_eye_calibrationstart) service for how to start a session with a working directory.
+The captures and robot poses are loaded from the provided directory. The directory is opened as read-only, and no new
+captures can be made during this session. However, calls to the
+[hand_eye_calibration/calibrate](#hand_eye_calibrationcalibrate) service can be made to compute the hand-eye transform
+from the loaded data.
+
+### infield_correction/read
+[zivid_interfaces/srv/InfieldCorrectionRead.srv](./zivid_interfaces/srv/InfieldCorrectionRead.srv)
+
+Returns the state of the [infield
+correction](https://support.zivid.com/en/latest/academy/camera/infield-correction.html) of the camera.
+
+### infield_correction/reset
+[std_srvs/srv/Trigger](https://docs.ros2.org/latest/api/std_srvs/srv/Trigger.html)
+
+Resets the infield correction on the camera to factory settings.
+
+### infield_correction/start
+[std_srvs/srv/Trigger](https://docs.ros2.org/latest/api/std_srvs/srv/Trigger.html)
+
+Prepares for infield correction, and clears any infield correction captures gathered so far in the `zivid_camera` node.
+
+This service must be called before gathering captures using the [infield_correction/capture](#infield_correctioncapture)
+service. However, other infield correction services can be used without calling the start service first. This service
+can also be used to restart an active infield correction session.
+
+### infield_correction/capture
+[zivid_interfaces/srv/InfieldCorrectionCapture.srv](./zivid_interfaces/srv/InfieldCorrectionCapture.srv)
+
+Takes a capture to be used for infield correction. Please point the camera at a Zivid infield calibration object. It is
+recommended to cover several distances, with one or more captures at each distance. Successful captures are stored in
+the `zivid_camera` node.
+
+Before calling this service, a call to [infield_correction/start](#infield_correctionstart) must have been made first.
+
+This service call will perform a relatively slow but high-quality point cloud capture with the connected camera. Any
+settings applied to the camera in the ROS driver will be ignored for calls to this service, appropriate settings are
+automatically used. The resulting point cloud and color image will be published just like during a normal call to the
+[capture](#capture) service.
+
+After sufficient number of captures, proceed by calling the [infield_correction/compute](#infield_correctioncompute)
+service to compute the camera correction based on the captures gathered so far. To compute the correction and write it
```

Comment:
I would write `camera correction and camera verification metrics` or something (check how we phrase it in Studio). Since now this service  (compute) is also what you use to verify the camera now.